invenio_search_ui/searchconfig.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Search app configuration helper."""
+
+from copy import deepcopy
+
+from flask import current_app
+
+
+class OptionsSelector:
```

Comment:
little naming police:
```suggestion
class SearchOptionsSelector:
```